docs/src/test/scala/docs/ApiMayChangeDocCheckerSpec.scala

```diff
@@ -0,0 +1,59 @@
+package docs
+
+import java.lang.reflect.Method
+
+import akka.annotation.ApiMayChange
+import org.reflections.Reflections
+import org.reflections.scanners.{ MethodAnnotationsScanner, TypeAnnotationsScanner }
+import org.reflections.util.{ ClasspathHelper, ConfigurationBuilder }
+import org.scalatest.{ Matchers, WordSpec }
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.io.Source
+
+class ApiMayChangeDocCheckerSpec extends WordSpec with Matchers {
+
+  def prettifyName(clazz: Class[_]): String = {
+    clazz.getCanonicalName.replaceAll("\\$minus", "-").split("\\$")(0)
```

Comment:
Converts the name to something listable, for example it strips the dollar suffixes